test/test_tleap.py

```diff
@@ -0,0 +1,246 @@
+"""
+Tests tleap tools.
+"""
+
+import warnings
+import numpy as np
+import logging as log
+import subprocess as sp
+import random as random
+import parmed as pmd
+import paprika
+from paprika.align import *
+from paprika.dummy import *
+from paprika.tleap import *
+import os
+import shutil
+import sys
+import filecmp
+import pytest
+
+
+def rm_solvated_files():
+    files = [
+        'solvate.pdb', 'solvate.prmtop', 'solvate.rst7', 'tleap_apr_solvate.in', 'leap.log', 'tmp.pdb',
+        'cb6-but-dum.tleap.in', 'cb6-but-dum.pdb', 'cb6-but-dum.prmtop', 'cb6-but-dum.rst7', 'tleap.in', 'dm1.mol2', 'dummy.frcmod',
+        'solvate.in', 'solvate.tleap.in', 'full.topo', 'full.crds', 'full.pdb'
+    ]
+    for f in files:
+        if os.path.isfile('./cb6-but/' + f):
+            os.remove('./cb6-but/' + f)
+
+@pytest.mark.slow
+def test_solvation_simple():
+    """ Test that we can solvate CB6-BUT using default settings. """
+    waters = np.random.randint(100, 10000)
+    #waters = 2922
+    log.debug('Trying {} waters with default settings...'.format(waters))
+    sys = System()
+    sys.template_file = 'cb6-but/tleap_solvate.in'
+    sys.output_path = 'cb6-but'
+    sys.loadpdb_file = 'cb6-but.pdb'
+    sys.buffer_target = waters
+    sys.output_prefix = 'solvate'
+    sys.build()
+    grepped_waters = sp.check_output(["grep -oh 'WAT' ./cb6-but/solvate.prmtop | wc -w"], shell=True)
+    assert int(grepped_waters) == waters
+    rm_solvated_files()
```

Comment:
I thought we were going to add options to *not* delete test files?

Reply:
Oh, hm, if the `assert` fails, maybe it won't if we run with the `pytest` options you sent me.

Reply:
I figured I'd reorganize the tests as a separate pull request.  So i just left them as mostly the same for now.

And yes, if we use `pytest -x` it seems to keep the output